src/status_im/ui/screens/wallet/request/views.cljs

```diff
@@ -36,34 +36,36 @@
       :size    256}]))
 
 (views/defview request-transaction []
-  [react/keyboard-avoiding-view  {:style wallet-styles/wallet-modal-container}
-   [status-bar/status-bar {:type :wallet}]
-   [toolbar-view]
-   [react/view styles/main-container
-    [react/scroll-view
-     [react/view styles/network-container
-      ;;TODO (andrey) name of active network should be used
-      [components/network-label styles/network-label "Testnet"]
-      [react/view styles/qr-container
-       [qr-code]]]]
-    [react/view styles/choose-wallet-container
-     [components/choose-wallet]]
-    [react/view styles/amount-container
-     [components/amount-input
-      {:input-options {:on-change-text
-                       #(re-frame/dispatch [:set-in [:wallet/request-transaction :amount] %])}}]
-     [react/view styles/choose-currency-container
-      [components/choose-currency styles/choose-currency]]]]
-   [react/view styles/separator]
-   [react/view styles/buttons-container
-    [vi/icon :icons/share {:color :white :container-style styles/share-icon-container}]
-    [react/text {:style      styles/button-text
-                 :font       (if platform/android? :medium :default)
-                 :uppercase? (get-in platform/platform-specific [:uppercase?])} (i18n/label :t/share)]
-    [react/view {:flex 1}]
-    [react/touchable-highlight {:on-press send-request}
-     [react/view styles/send-request-container
+  (views/letsubs [scroll (atom nil)]
```

Comment:
Can you add some comment detailing what this scroll logic is doing?

Reply:
sure